ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -152,6 +152,9 @@ public class ConfigurationApplication implements SparkApplication {
 
   private Integer exportResultLimit = 1000;
 
+  private List<String> requiredExportAttributes =
+      ImmutableList.of("title", "security.classification", "security.releasability");
```

Comment:
I may be misremembering but is it possible to create a metacard type that doesn't include the "security" attributes? If that's the case we probably don't want to include them in this default list.